### PR TITLE
[0.6] Correcting GOES 2 Operational time range

### PR DESCRIPTION
**Pull request by DanRyanIrish**
This PR backports PR #1319 to the version 0.6 branch.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -67,6 +67,7 @@
  * MapCube.plot and MapCube.peek now support a user defined plot_function argument for customising the animation.
  * Added new sample data file, an AIA cutout file.
  * Moved documentation build directory to doc/build
+ * Changed start of GOES 2 operational time range back to 1980-01-04 so data from 1980 can be read into GOESLightCurve object
```
Comment (Cadair) [inline CHANGELOG.md]: This line should be under the 0.6.2 heading at the top.

**Comment by Cadair** [comment]
Thanks @DanRyanIrish

File: CHANGELOG.md (hunk 1 above)
Comment (DanRyanIrish) [inline CHANGELOG.md]: Is this the only problem with the PR or have you not looked at it all yet?

**Comment by DanRyanIrish** [comment]
You're welcome @Cadair.

File: CHANGELOG.md (hunk 1 above)
Comment (Cadair) [inline CHANGELOG.md]: Seeing how it's a copy of the one to master the rest of it is ok.

**Comment by Cadair** [comment]
:+1: pending CI

**Comment by dpshelio** [comment]
:+1:

**Pull request merged**